instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/cassettes/test_async_chat_completion_404.yaml

```diff
@@ -0,0 +1,73 @@
+interactions:
+- request:
+    body: '{"messages": [{"role": "user", "content": "Say this is a test"}], "model":
+      "this-model-does-not-exist"}'
+    headers:
+      accept:
+      - application/json
+      accept-encoding:
+      - gzip, deflate
+      connection:
+      - keep-alive
+      content-length:
+      - '103'
+      content-type:
+      - application/json
+      host:
+      - api.openai.com
+      user-agent:
+      - AsyncOpenAI/Python 1.26.0
+      x-stainless-arch:
+      - arm64
+      x-stainless-async:
+      - async:asyncio
+      x-stainless-lang:
+      - python
+      x-stainless-os:
+      - MacOS
+      x-stainless-package-version:
+      - 1.26.0
+      x-stainless-runtime:
+      - CPython
+      x-stainless-runtime-version:
+      - 3.12.5
+    method: POST
+    uri: https://api.openai.com/v1/chat/completions
+  response:
+    body:
+      string: "{\n    \"error\": {\n        \"message\": \"The model `this-model-does-not-exist`
```

Comment:
Have you recorded this before recent changes to conftest?

Reply:
opps, will adjust now

Reply:
I think this is the problem, maybe someone can add the skip-changelog tag on it and merge. Then, this PR can be rebased over it. https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2989